adafruit_max31856.py

```diff
@@ -208,7 +217,7 @@ def noise_rejection(self) -> int:
         return 60
 
     @noise_rejection.setter
-    def noise_rejection(self, frequency: int):
+    def noise_rejection(self, frequency: int) -> None:
```

Comment:
You actually annotated the return correctly, but while we're editing this file, let's make the getter and setter for `noise_rejection` not be `int` but rather `Literal[50, 60]`!

Reply:
rats! missed that in the docstring!